content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+
+If you run your mesh on a single cluster, a single control plane deployment
+suffices. Each workload instance in the cluster connects to a control plane
+instance to get its configuration.
+
+{{< image width="50%"
+    link="./single-cluster.svg"
+    alt="A service mesh with a single control plane"
+    title="Single control plane"
+    caption="A service mesh with a single control plane"
+    >}}
+
+### Multiple control planes
+
+You can deploy multiple control planes across the mesh; such a configuration is
+known as a **multi-control plane**.
+
+Multiple control planes afford improved performance and availability for meshes
+that span [multiple clusters](#multiple-clusters) across multiple zones and
+regions.
+
+Istio supports control plane deployments at various levels of availability:
+
+- Control plane per region: *lowest availability*
+- Control plane per zone
+- Control plane per cluster *highest availability*
+
+The configuration of each control plane could be identical or different
+depending on the [network](#network-models) and [cluster](#cluster-models)
+models of the system.
+
+{{< image width="50%"
+    link="./multi-control.svg"
+    alt="A service mesh with multiple control planes"
+    title="Multiple control planes"
+    caption="A service mesh with multiple control planes"
+    >}}
+
+### Managed control plane
+
+Many cloud providers offer a control plane, which they manage on behalf of their
+customer.
+
+Typically, managed control planes guarantee some level of performance and
+availability, which you can assume when building your system. Managed control
+planes affords the following benefits:
+
+- Greatly reduce the complexity of user deployments
+- Greatly reduce system administration costs
+- Effectively eliminate the usage costs associated with the control plane:
+    - CPU
+    - Memory
+    - Network
+
+{{< image width="50%"
+    link="./managed-control.svg"
+    alt="A service mesh with a managed control plane"
+    title="Managed control plane"
+    caption="A service mesh with a managed control plane"
+    >}}
+
+## Identity and trust models
+
+When a workload instance is created within a service mesh, Istio assigns the
+workload an identity. An identity is a unique name backed by a key material, for
+example:
+
+- [X.509 certificate](https://en.wikipedia.org/wiki/X.509)
+- [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)
+
+Each identity Istio assigns is understood throughout the mesh and you can use it
+to enable mutual authentication and enforce policies.
+
+An identity contains a substring identifying the mesh that created it. For
+example in `spiffe://mytrustdomain.com/ns/default/sa/myname` the substring
+identifying the mesh is: `mytrustdomain.com`. We call this substring the {{<
+gloss>}}trust domain{{< /gloss >}} of the mesh. Every Istio mesh has a globally
+unique trust domain used to create identity names.
+
+The Certificate Authority (CA) creates the identities and their key material.
```

Comment:
The CA signs the identities, it doesn't create them. The identities are created through k8s api server today (by creating a service account).

Reply:
@smawson I'll change the text to reflect that nuance. Do you think I should include the fact about how the identities are created of would that be a k8s-specific implementation detail?

Reply:
Yeah that's maybe overly K8s specific, although if we define an identity as a namespace/service account that's also k8s specific. We haven't really discussed non-k8s identities enough (even though we can in theory support them).

Maybe we can make this somewhat generic and just say:

The Certificate Authority (CA) is responsible for creating and signing the certificates used to attest identities used within the mesh.

Actually that's a bad sentence so please rewrite, but you get the idea :)